samples/matter/common/src/persistent_storage/Kconfig

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if NCS_SAMPLE_MATTER_PERSISTENT_STORAGE
+
+config NCS_SAMPLE_MATTER_SETTINGS_STORAGE_BACKEND
+	bool "Settings based storage implementation for Matter samples"
+	default y
+
+config NCS_SAMPLE_MATTER_SECURE_STORAGE_BACKEND
+	bool "Secure storage implementation for Matter samples"
+	select TRUSTED_STORAGE
+	select PSA_PROTECTED_STORAGE
+	default n
```

Comment:
Why about making here `select` to choose the backend? I mean `secure` and `raw settings`.